src/assignments/hello-world.rst

```diff
@@ -1,4 +1,65 @@
-.. _hello-world:
+.. _assignment0:
 
 Assignment #0: Hello, World!
-============================
\ No newline at end of file
+============================
+
+The purpose of this initial assignment is to familiarize yourself with the process of running the autograding tests and submitting your work via GitHub Classroom. Even if you are familiar with GitHub Classroom, this assignment will contain several new items that are specific to working with Java assignments.
+
+Getting Started
+---------------
+
+Let's set up the assignment on our computer and learn about its basic structure.
+
+First, find Assignment #0 in Canvas and click on the invitation link. After accepting, go to your assignment repository on GitHub.
+
+Follow the instructions to :ref:`clone your repository <clone-csharp-project>` and open it in Visual Studio.
+
+When the solution opens, check out what the repository contains in the *Solution Explorer* on the left. You will find two projects within the solution.
+
+Open up the ``SayHelloClass`` class in the ``HelloWorldAutograded`` project. This is where you will work on your code.
+
+Now, turn your attention to ``Program.cs`` in the ``HelloWorldAutograded`` project. We have given you some code that will output the string returned by the ``SayHello()`` function in ``SayHelloClass``.
+
+To run the program, hit the *Run* button at the top right-hand corner of the window or right-click on the ``HelloWorldAutograded`` project and select *Run Project*.
```

Comment:
In Windows, they need to click on the solid green triangle next to the name of their project.  They can also press F5 to run their project.  

(I can share images if we need them).

Reply:
Thanks!